assets/src/scripts/components/dailyValueHydrograph/selectors/time-series-data.js

```diff
@@ -0,0 +1,151 @@
+import {DateTime} from 'luxon';
```

Comment:
This was time-series-line.js. I added new selectors for returning information about the points at the cursor offset (getCursorEpochTime, getDataAtCursor, getCursorPoint